ARM/ARM-Template-Deployment.yml

```diff
@@ -0,0 +1,55 @@
+name: workflow to deploy an ARM Template using CLI commands
+on: push
+
+# CONFIGURATION
+# For help, go to https://github.com/Azure/Actions
+#
+# Set up the following secrets in your repository:
+#   AZURE_CREDENTIALS
+# Change these variables for your configuration:
+#   AZURE_RESOURCE_GROUP, AZURE_LOCATION
+
+env:
+  AZURE_RESOURCE_GROUP: ActionsDemo   # set this to your Azure Resource group's name  
+  AZURE_LOCATION: WestEurope  # set this to your Location Name 
+  
+jobs:
+  build-and-deploy-to-dev:
+    runs-on: ubuntu-latest
+    steps:
+                
+    # Authentication
+    - name: Azure Login
+      uses: azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS  }}
+    
+    # Checkout
+    - name: Checkout
+      uses: actions/checkout@v1
+        
+    # Deployment of template
+    - name: Deploy ARM Template
+      uses: azure/CLI@v1
+      with:
+        azcliversion: latest
+        inlineScript: |
+          az group create --location ${{ env.AZURE_LOCATION }} --name ${{ env.AZURE_RESOURCE_GROUP }}
```

Comment:
Add a comment that variables need to be set as env variables. Also provide a GitHub document link to the same.